tests/LinkedListTest.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gtest/gtest.h>
+#include "LinkedList.cpp"
+
+TEST(LinkedListTest, InstantiateEmptyList) {
```

Comment:
are we exposing 3 tests for students?

Reply:
I'm thinking of showing test results from the 3 test C++ files the students submit (regardless of what they add/modify), but they will have 0 weight.

The final score will be graded on the test cases written by us only. (We don't use their `tests/*.cpp` when running our grader.)

What do you think?

Reply:
totally agree